### ISSUE TITLE
Consider supporting yank

### ISSUE THREAD

**Issue by mcwoodring** [d-e-s-o/cargo-http-registry#43]
This project is super helpful for, among other things, safely learning/demonstrating how to publish crates - thank you.

If it supported **yank** in addition to **publish**, the complete lifecycle could be safely experimented with/demonstrated. Any plans to support that? If not, I'm happy to take a run at it you have any suggestions for how you'd approach it.

**Comment by d-e-s-o** [comment]
Makes sense. I have no plans for such support, but happy to accept pull requests. I think the reference is https://doc.rust-lang.org/cargo/reference/registry-web-api.html#yank. If you decide to work on it, please make sure to include tests and stick to the existing coding style.
